### PR TITLE
Run generate before prerelease

### PR DESCRIPTION
**Pull request by fabianvf**
**Description of the change:**
`make generate` is now run before `make prerelease`, and the documentation is updated to match.

**Motivation for the change:**
This is a necessary step in the release process.

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [ ] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [ ] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

**Review by estroz — APPROVED** [review]
/lgtm

File: website/content/en/docs/contribution-guidelines/releasing.md [hunk 1]
```diff
@@ -76,15 +76,17 @@ make prerelease
 
 The following changes should be present:
 
+- `Makefile`: IMAGE_VERSION should be modified to the upcoming release tag. (This variable ensures sampleprojects have been tagged correctpy prior to the release commit.)
```
Comment (jmrodri) [inline website/content/en/docs/contribution-guidelines/releasing.md:79]:
```suggestion
- `Makefile`: IMAGE_VERSION should be modified to the upcoming release tag. (This variable ensures sampleprojects have been tagged correctly prior to the release commit.)
```

File: website/content/en/docs/contribution-guidelines/releasing.md [hunk 2]
```diff
@@ -76,15 +76,17 @@ make prerelease
...
 - `changelog/fragments/*`: commit deleted fragment files (deleted by changelog generation).
 - `website/content/en/docs/upgrading-sdk-version/v1.3.0.md`: commit changes (created by changelog generation).
 - `website/config.toml`: commit changes (modified by release script).
+- `testdata/*`: Generated sample code.
 
 Commit these changes and push to your remote (assuming your remote is named `origin`):
 
 ```sh
-git add --all
+git add Makefile changelog website testdata
```
Comment (jmrodri) [inline website/content/en/docs/contribution-guidelines/releasing.md:89]: +1

File: website/content/en/docs/contribution-guidelines/releasing.md [hunk 3]
```diff
@@ -186,7 +185,7 @@ All of the following changes should be present (and no others).
 Commit these changes and push these changes **to your fork**:
 
 ```sh
-git add --all
+git add Makefile changelog website testdata
```
Comment (jmrodri) [inline website/content/en/docs/contribution-guidelines/releasing.md:188]: +1

**Review by jmrodri — CHANGES_REQUESTED** [review]
Looks just fine except for the typo.

**Comment by jmrodri** [comment]
Fixing typo will allow me to approve this.

**Review by jmrodri — APPROVED** [review]
/lgtm

**Pull request merged**